extensions/binding/org.eclipse.smarthome.binding.digitalstrom/src/main/java/org/eclipse/smarthome/binding/digitalstrom/handler/BridgeHandler.java

```diff
@@ -292,6 +363,12 @@ private Config loadAndCheckConnectionData(Configuration thingConfig) {
     @Override
     public void dispose() {
         logger.debug("Handler disposed");
+        if (reconnectTracker != null && !reconnectTracker.isCancelled()) {
+            reconnectTracker.cancel(true);
+        }
+        if (eventListener != null) {
+            eventListener.stop();
```

Comment:
Because by each little disconnect this method will be called, we only wanted to stop the services and do not delete it. So they can resume their work after a reconnect.

By the way, then the other services would be set to null, too.